### ISSUE TITLE
panic: runtime error: invalid memory address or nil pointer dereference

### ISSUE THREAD

**Issue by npyzza** [hashicorp/terraform-provider-azurerm#13460]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform (and AzureRM Provider) Version
```
Terraform v1.0.6
on linux_amd64
- Installing hashicorp/external v2.1.0...
- Installed hashicorp/external v2.1.0 (signed by HashiCorp)
- Installing hashicorp/azurerm v2.75.0...
- Installed hashicorp/azurerm v2.75.0 (signed by HashiCorp)
```
### Affected Resource(s)

* `azurerm_app_service_managed_certificate`

### Terraform Configuration Files

https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs

## Panic Output
```
Stack trace from the terraform-provider-azurerm_v2.75.0_x5 plugin:

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x552940a]

goroutine 952 [running]:
github.com/hashicorp/terraform-provider-azurerm/internal/services/web.resourceAppServiceManagedCertificateRead(0xc002780180, 0x5935640, 0xc0015dd880, 0x0, 0x0)
	/opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-azurerm/internal/services/web/app_service_managed_certificate_resource.go:221 +0x6ea
github.com/hashicorp/terraform-provider-azurerm/internal/services/web.resourceAppServiceManagedCertificateCreateUpdate(0xc002780180, 0x5935640, 0xc0015dd880, 0x0, 0x0)
	/opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-azurerm/internal/services/web/app_service_managed_certificate_resource.go:192 +0xd5d
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).create(0xc001363c00, 0x69098e8, 0xc00100ce80, 0xc002780180, 0x5935640, 0xc0015dd880, 0x0, 0x0, 0x0)
	/opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-azurerm/vendor/github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema/resource.go:318 +0x1ee
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).Apply(0xc001363c00, 0x69098e8, 0xc00100ce80, 0xc0004ddab0, 0xc001183b20, 0x5935640, 0xc0015dd880, 0x0, 0x0, 0x0, ...)
	/opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-azurerm/vendor/github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema/resource.go:456 +0x67b
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*GRPCProviderServer).ApplyResourceChange(0xc001075620, 0x69098e8, 0xc00100ce80, 0xc001bb1f90, 0xc00100ce80, 0x5e29060, 0xc00179b000)
	/opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-azurerm/vendor/github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema/grpc_provider.go:955 +0x8ef
github.com/hashicorp/terraform-plugin-go/tfprotov5/server.(*server).ApplyResourceChange(0xc001cb8be0, 0x6909990, 0xc00100ce80, 0xc0004dd960, 0xc001cb8be0, 0xc00179b0b0, 0xc0027f2ba0)
	/opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-azurerm/vendor/github.com/hashicorp/terraform-plugin-go/tfprotov5/server/server.go:332 +0xb5
github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5._Provider_ApplyResourceChange_Handler(0x5e29060, 0xc001cb8be0, 0x6909990, 0xc00179b0b0, 0xc0016ea900, 0x0, 0x6909990, 0xc00179b0b0, 0xc001507800, 0x3fc)
	/opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-azurerm/vendor/github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5/tfplugin5_grpc.pb.go:380 +0x214
google.golang.org/grpc.(*Server).processUnaryRPC(0xc000226a80, 0x6958398, 0xc000103200, 0xc002c87440, 0xc000c10ab0, 0x9fa22e0, 0x0, 0x0, 0x0)
	/opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-azurerm/vendor/google.golang.org/grpc/server.go:1292 +0x52b
google.golang.org/grpc.(*Server).handleStream(0xc000226a80, 0x6958398, 0xc000103200, 0xc002c87440, 0x0)
	/opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-azurerm/vendor/google.golang.org/grpc/server.go:1617 +0xd0c
google.golang.org/grpc.(*Server).serveStreams.func1.2(0xc001043020, 0xc000226a80, 0x6958398, 0xc000103200, 0xc002c87440)
	/opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-azurerm/vendor/google.golang.org/grpc/server.go:940 +0xab
created by google.golang.org/grpc.(*Server).serveStreams.func1
	/opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-azurerm/vendor/google.golang.org/grpc/server.go:938 +0x1fd

Error: The terraform-provider-azurerm_v2.75.0_x5 plugin crashed!

This is always indicative of a bug within the plugin. It would be immensely
helpful if you could report the crash with the plugin's maintainers so that it
can be fixed. The output above should help diagnose the issue.
```

### Expected Behaviour

```
Plan: 20 to add, 0 to change, 0 to destroy.
```

### Actual Behaviour

```
Error: Plugin did not respond
│ 
│   with module.core.azurerm_app_service_managed_certificate.this,
│   on modules/core/dns.tf line 41, in resource "azurerm_app_service_managed_certificate" "this":
│   41: resource "azurerm_app_service_managed_certificate" "this" {
│ 
│ The plugin encountered an error, and failed to respond to the
│ plugin.(*GRPCProvider).ApplyResourceChange call. The plugin logs may
│ contain more details.
╵
```

Ran the apply again and got this:
```
│ Error: A resource with the ID "xxxx" already exists - to be managed via Terraform this resource needs to be imported into the State. Please see the resource documentation for "azurerm_app_service_managed_certificate" for more information.
│ 
│   with module.core.azurerm_app_service_managed_certificate.this,
│   on modules/core/dns.tf line 41, in resource "azurerm_app_service_managed_certificate" "this":
│   41: resource "azurerm_app_service_managed_certificate" "this" {
│ 
╵
```

This has happened both in local and remote states.

It is creating the resource but crashing before it is added to state.

### Steps to Reproduce

1. `terraform apply`

### References

https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs

**Comment by aristosvo** [comment]
Hi @npyzza! This is a duplicate of #13353 and solved in the latest version of the provider, v2.77.0

**Comment by tombuildsstuff** [comment]
Duplicate of #13353 / fixed in v2.77.0 - thanks @aristosvo :)
